src/test/kotlin/step2/calculator/OperatorTest.kt

```diff
@@ -12,29 +12,39 @@ import step2.calculator.Operator.Companion.INVALID_OPERATOR_ERROR_MESSAGE
 import java.util.stream.Stream
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
-@DisplayName("계산을 담당하는 Operator 테스트")
+@DisplayName("계산을 담당하는 연산자 객체인 Operator 테스트")
 internal class OperatorTest {
-    @DisplayName("주어진 연산자가 올바른 경우 연산자 매핑 테스트")
+    @DisplayName("주어진 연산자가 올바른 경우 getOperator 함수를 호출하면 올바른 Operator 반환")
```

Comment:
```suggestion
    @DisplayName("주어진 연산자가 올바른 경우 올바른 Operator 반환")
```
getOperator 함수를 호출하면 이라는 문구는 없어도 될 것 같아요
그런데 올바른 Operator는 무엇을 의미하는 걸까요?

Reply:
https://github.com/next-step/kotlin-racingcar/pull/361#discussion_r744846517